src/GenAPI/Microsoft.DotNet.GenAPI.Shared/Microsoft.DotNet.GenAPI.Shared.csproj

```diff
@@ -0,0 +1,23 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
```

Comment:
This project isn't really a Visual Studio Shared project. Instead, it's just a class library. Only use the "Shared" name suffix if it's a VS Shared project. I think it makes sense to keep this a class library and rename the project and folder from `Microsoft.DotNet.GenAPI.Shared` to `Microsoft.DotNet.GenAPI.Core`.